src/boot.php

```diff
@@ -9,16 +9,29 @@
 
 require_once 'shortcode.php';
 require_once 'widget.php';
+require_once 'settings-page.php';
+require_once 'html/banner-display.php';
 
 /**
  * Initialises the app.
  */
 function boot() {
+
 	add_shortcode( 'south-african-covid-19-banner', 'SeagynDavis\SouthAfricaCovid19Banner\Shortcode\shortcode' );
+
 	add_action(
 		'widgets_init',
 		function() {
 			register_widget( 'SeagynDavis\SouthAfricaCovid19Banner\Widget\Banner' );
 		}
 	);
+
+	add_action( 'admin_init', 'SeagynDavis\SouthAfricaCovid19Banner\SettingsPage\settings_init' );
+
+ 	add_action( 'admin_menu', 'SeagynDavis\SouthAfricaCovid19Banner\SettingsPage\options_page' );
+
+	add_action( 'wp_head', 'SeagynDavis\SouthAfricaCovid19Banner\BannerDisplay\header_banner' );
```

Comment:
I don't think this will have the desired effect because this is usually in `<head>`. Perhaps just rely on the CSS to position it?